app/views/diaper_drive_participants/new_modal.js.erb

```diff
@@ -1,2 +1,2 @@
-$("#modal_new_diaper_drive").html("<%= j (render "diaper_drive_participants/new_modal") %>");
-$("#modal_new_diaper_drive").modal("show");
\ No newline at end of file
+$("#modal_new").html("<%= j (render "diaper_drive_participants/new_modal") %>");
+$("#modal_new").modal("show");
```

Comment:
Nice! 

Am I understanding that this modal can now be used more generically, and filled with whatever we need? Very cool.